src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -16,6 +17,14 @@
           Condition="'$(RestorePackages)'=='true' AND '$(IsTestProject)' == 'true' AND '$(TestRuntimeProjectJson)' != ''">
     <Exec Command="$(DnuRestoreCommand) &quot;$(TestRuntimeProjectJson)&quot;" StandardOutputImportance="Low" CustomErrorRegularExpression="^Unable to locate .*" />
   </Target>
+  
+  <Target Name="ValidateTestRuntimeDependencies"
+          AfterTargets="RestoreTestRuntimePackage"
```

Comment:
Unlike the ToolRuntime it doesn't look like we have a semaphore to ensure we don't run the RestoreTestRuntimePackage more than once. We should consider fixing that and when we do we should also do the ValidateTestRuntimeDependencies at the same time. I'm fine with this being here but just want to be sure that it doesn't run for every test project.